lib/codegen.js

```diff
@@ -251,6 +252,18 @@ function enhanceCode(source, opts, type){
     } else {
         return source;
     }
+
+    if (opts.tslintFix === undefined || opts.tslintFix === true) {
```

Comment:
`opts.tslintFix === undefined` should probably be considered as not wanting to run tslintfix